tsa-test/src/test/resources/checkers/transaction-rollback/sender.fc

```diff
@@ -0,0 +1,16 @@
+#include "../../imports/stdlib.fc";
+#include "../../imports/tsa_functions.fc";
+
+() on_internal_message_send(int balance, int msg_value, cell in_msg_full, slice msg_body, int input_id) impure method_id {
+
+}
+
+() main() impure {
+    var data1 = tsa_get_c4(1).begin_parse();
+    var int1 = data1~load_uint(1);
+    throw_if(257, (int1 != 0));
+    tsa_send_internal_message(1, 0);
+    var data2 = tsa_get_c4(1).begin_parse();
+    var int2 = data2~load_uint(1);
+    throw_if(258, (int2 != 0));
```

Comment:
Isn't this supposed to be `throw_unless`? We expect `int2` to be zero, because the transaction should have been rolled back